backend/models/teamspaceSetting.js

```diff
@@ -101,15 +101,16 @@ class TeamspaceSettings {
 				return Promise.reject(responseCodes.FILE_FORMAT_NOT_SUPPORTED);
 			}
 
+			const Mitigation = require("./mitigation");
+			const readCSVProm = await Mitigation.importCSV(account, file);
+
 			const storeFileProm = FileRef.storeMitigationsFile(account, username, filename, file).then(async () => {
```

Comment:
If `readCSVProm` fail, does it still update the csv with the broken one? surely we don't want to do that?

Reply:
@carmenfan When importCSV throws an error, it doesn't seem to proceed to storeMitigationsFile